### PR TITLE
Pass portion_sizes to SingleHopperDriver

### PR DESCRIPTION
**Pull request by pramati**
Updates the constructor call to user portion_sizes.

### CODE REVIEW

File: miso_peripherals/src/miso_peripherals/hopper/sim_driver.py [hunk 1]
```diff
@@ -92,7 +92,12 @@ class SimSingleHopperDriver(SingleHopperDriver):
 
     def __init__(self,  dispenser):
         """Construct a new SimSingleHopperDriver."""
-        super(SimSingleHopperDriver, self).__init__()
+        portion_sizes = {
```
Comment (luke-powers) [inline miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:95]: Should this be picked up from the world.launch parameters?

Comment (rsinnet) [reply miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:95]: These do not need to be the same. But should make sense given the default fill size of the hopper.

Comment (pramati) [reply miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:95]: Yeah perhaps we could refactor, but just added some default value here so we dont forget to add it in the base branch.

File: miso_peripherals/nodes/ram_hopper_simulator_node [hunk 2]
```diff
@@ -3,6 +3,11 @@
 
 from miso_peripherals.hopper.controller import SimHopperController
 from miso_peripherals.hopper.host import HopperControllerNode
+from miso_peripherals.hopper.sim_driver import (
```
Comment (rsinnet) [inline miso_peripherals/nodes/ram_hopper_simulator_node:6]: It was working before this, no?

File: miso_peripherals/src/miso_peripherals/hopper/sim_driver.py (hunk 1 above)
Comment (rsinnet) [reply miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:95]: A TODO seems appropriate here.

**Review by rsinnet — APPROVED** [review]

**Comment by rsinnet** [comment]
Tests need to be updated or something here is wrong.

**Comment by rsinnet** [comment]
I guess this is invalid now. I have #61 now.

**Pull request closed without merging**